hack/dind-cluster.sh

```diff
@@ -94,6 +95,15 @@ function start() {
   echo "OPENSHIFT_ADDITIONAL_ARGS='${additional_args}'" > "${config_root}/additional-args"
   copy-runtime "${origin_root}" "${config_root}/"
 
+  ovn_kubernetes=
+  if [[ -d "${ovn_root}" ]]; then
+    copy-ovn-runtime "${ovn_root}" "${config_root}/"
+    ovn_kubernetes=1
+  else
+    ovn_kubernetes=
```

Comment:
redundant, you already set it before the `if`

Reply:
fixed